Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md

```diff
@@ -0,0 +1,136 @@
+# Center items using Flexbox
+
+CSS Flexbox has been out there for some time and given that you know about all its features, it could be a boon and save you a lot of time trying to align elements and making them responsive.
+
+**Today we will see how to center elements using flexbox in just 3 lines of code!!**
+
+First lets take a container and add a box inside it.
+
+```html
+<div class="container">
+    <div class="box"></div>
+</div>
+```
+
+```css
+.box{
+    height: 50px;
+    width: 50px;
+    background-color: red;
+}
+
+.container{
+    border: 2px solid black;
+    height: 200px;
+}
+```
+
+![Initial box and container](https://user-images.githubusercontent.com/86849664/139646837-0af823d2-bf6b-42e2-b941-eb02480533ec.png)
+
+In order to center the inner div we need to make the parent div a flex container.
+
+To do so, we will use `display: flex` property for the parent container. This will make all the elements inside the container **flex-items**.
+
+```css
+.container{
+    border: 2px solid black;
+    height: 200px;
+    display: flex;
+}
+```
+
+
+
+## Centering horizontally
+
+To center our item horizontally we use the `justify-content: center;`, this will center our block along the main-axis. The direction of main-axis could be changed using the `flex-direction` property.
+
+```css
+.container{
+    border: 2px solid black;
+    height: 200px;
+    display: flex;
+    justify-content: center;
+}
+```
+
+![Horizontally centered](https://user-images.githubusercontent.com/86849664/139647071-6648f42f-9819-4134-845a-b3b818f9dd08.png)
+
+
+## Centering Vertically
+
+To center our item vertically we use the `align-items: center;`, this will center our block along the cross-axis. 
+
+```css
+.container{
+    border: 2px solid black;
+    height: 200px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+```
+
+![Vertically centered](https://user-images.githubusercontent.com/86849664/139647149-3a1a9b33-b92c-4abc-a8fe-8a5489a81dea.png)
+
+
+The nice aspect of flexbox is the styles apply to all the children of our flex container. If we add two more elements, they all stay centered within the parent container.
+
+```html
+ <div class="container">
+        <div class="box"></div>
+        <div class="box"></div>
+        <div class="box"></div>
+    </div>
+```
+
+![multiple boxes](https://user-images.githubusercontent.com/86849664/139647215-ceb290d0-489e-405f-bec0-0e38b93bf363.png)
+
+
+
+## flex-direction property
+
+By default the flex-direction is set as row. If we change the flex-direction to column that would interchange the main-axis and the cross-axis. It will also make our items stack in a column instead of a row like before.
+
+![flex-direction](https://user-images.githubusercontent.com/86849664/139647286-56adde37-e110-407f-b338-30754270d768.png)
+
+<p align="center"> Image Source: https://miro.medium.com/max/700/1*bw4hK0MYxFXbw7ylW2fR5Q.png </p>

```

Comment:
Embed the link in the website name.

Reply:
@kashishahuja2002 I've made the changes pls check.